ProjFS.Mac/PrjFSKextTests/HandleOperationTests.mm

```diff
@@ -669,6 +669,20 @@ - (void) testDeleteDirectoryWithDisappearingVirtualizationRoot {
     XCTAssertTrue(MockCalls::CallCount(ProviderMessaging_TrySendRequestAndWaitForResponse) == 1);
 }
 
+- (void) testDeleteDirectoryWhenFirstRequestFails {
```

Comment:
Could you add some comments to make it a bit clearer what's being validated?

I think we're validating that when processing `KAUTH_VNODE_DELETE` the kext will not send additional messages if the first `TrySendRequestAndWaitForResponse` fails, but a comment here would be good.